### ISSUE TITLE
MSFT_SPInstallLanguagePack: Get-TargetResource does not detect already installed Norwegian Language Pack

### ISSUE THREAD

**Issue by andrstor** [PowerShell/SharePointDsc#1026]
#### Details of the scenario you tried and the problem that is occurring
I am using SharepointDSC 3.1.0.0 to install the norwegian language pack. The installation works fine, but will fail on the next run because it can't detect that the specific norwegian language pack is already installed.

#### Verbose logs showing the problem

VERBOSE: [astorfarm2]: LCM:  [ Start  Resource ]  [[SPInstallLanguagePack]Instal
lLPBinaries]
VERBOSE: [astorfarm2]: LCM:  [ Start  Test     ]  [[SPInstallLanguagePack]Instal
lLPBinaries]
VERBOSE: [astorfarm2]:                            [[SPInstallLanguagePack]Instal
lLPBinaries] Testing install status of SharePoint Language Pack
VERBOSE: [astorfarm2]:                            [[SPInstallLanguagePack]Instal
lLPBinaries] Getting install status of SharePoint Language Pack
VERBOSE: [astorfarm2]:                            [[SPInstallLanguagePack]Instal
lLPBinaries] Update is for the Norwegian Bokmål language
VERBOSE: [astorfarm2]:                            [[SPInstallLanguagePack]Instal
lLPBinaries] Language Pack Norwegian Bokmål is NOT found
VERBOSE: [astorfarm2]: LCM:  [ End    Test     ]  [[SPInstallLanguagePack]Instal
lLPBinaries]  in 0.3750 seconds.
VERBOSE: [astorfarm2]: LCM:  [ Start  Set      ]  [[SPInstallLanguagePack]Instal
lLPBinaries]
VERBOSE: [astorfarm2]:                            [[SPInstallLanguagePack]Instal
lLPBinaries] Setting install status of SharePoint Language Pack
VERBOSE: [astorfarm2]:                            [[SPInstallLanguagePack]Instal
lLPBinaries] No BinaryInstallDays specified, Update can be ran on any day.
VERBOSE: [astorfarm2]:                            [[SPInstallLanguagePack]Instal
lLPBinaries] No BinaryInstallTime specified, Update can be ran at any time. Star
ting update.
VERBOSE: [astorfarm2]:                            [[SPInstallLanguagePack]Instal
lLPBinaries] Writing install config file
VERBOSE: [astorfarm2]:                            [[SPInstallLanguagePack]Instal
lLPBinaries] Beginning installation of the SharePoint Language Pack
VERBOSE: [astorfarm2]: LCM:  [ End    Set      ]  [[SPInstallLanguagePack]InstallL
PBinaries]  in 225.6170 seconds.
PowerShell DSC resource MSFT_SPInstallLanguagePack  failed to execute 
Set-TargetResource functionality with error message: SharePoint Language Pack 
install failed, exit code was 1618 
    + CategoryInfo          : InvalidOperation: (:) [], CimException
    + FullyQualifiedErrorId : ProviderOperationExecutionFailure
    + PSComputerName        : localhost

#### Suggested solution to the issue
In MSFT_SPInstallLanguagePack.psm1 add norwegian as a check when extracting the english name of the language code. 

```powershell
if ($englishProducts -contains $languageEnglish -eq $true)
{
...
}
```
Fails because the english product name is "Norwegian" while

```powershell
$cultureInfo.EnglishName
```
gets the value "Norwegian Bokmål"

This can be fixed by extracting the english name of the language code like this:

```powershell
    switch ($cultureInfo.EnglishName)
    {
        "Dari (Afghanistan)" { $languageEnglish = "Dari" }
        "Chinese (Simplified, China)" { $languageEnglish = "Chinese (PRC)" }
        "Chinese (Traditional, Taiwan)" { $languageEnglish = "Chinese (Taiwan)" }
        "Portuguese (Brazil)" { $languageEnglish = "Portuguese (Brasil)" }
        "Portuguese (Portugal)" { $languageEnglish = "Portuguese (Portugal)" }
        "Serbian (Cyrillic, Serbia)" { $languageEnglish = "Serbian (Cyrillic)" }
        "Serbian (Latin, Serbia)" { $languageEnglish = "Serbian (Latin)" }
	"Norwegian Bokmål (Norway)" { $languageEnglish = "Norwegian" } #ADDED THIS
        Default {
            if ($cultureInfo.EnglishName -match "(\w*,*\s*\w*) \([^)]*\)")
            {
                $languageEnglish = $matches[1]
                $updateLanguage  = $matches[0]
                if ($languageEnglish.contains(","))
                {
                    $languages = $languageEnglish.Split(",")
                    $languageEnglish = $languages[0]
                }
            }  
        }
    }
```
#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
        SPInstallLanguagePack InstallLPBinaries
        {
            BinaryDir  = $sharepointLanguagePackPath
            Ensure     = "Present"
            PsDscRunAsCredential = $sPInstall
        }

```

#### The operating system the target node is running

OsName               : Microsoft Windows Server 2019 Datacenter
OsOperatingSystemSKU : DatacenterServerEdition
OsArchitecture       : 64-bit
WindowsVersion       : 1809
WindowsBuildLabEx    : 17763.1.amd64fre.rs5_release.180914-1434
OsLanguage           : en-US
OsMuiLanguages       : {en-US}

#### Version of SharePoint that is used (e.g. SharePoint 2016)

SharePoint 2016
#### Version and build of PowerShell the target node is running

PSVersion                      5.1.17763.134
#### Version of the DSC module that was used ('dev' if using current dev branch)
3.1.0.0

**Comment by ykuijs** [comment]
Have been able to reproduce this issue. Will correct this in my next bugfix PR

**Comment by ykuijs** [comment]
BTW: Thanks for the extensive explanation of the issue. Makes reproducing the issue much easier 👍

**Comment by ykuijs** [comment]
Fixed in PR #1036
